packages/connect-form/src/advanced-connection-options.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { Label, spacing } from '@mongodb-js/compass-components';
+import { css } from '@emotion/css';
+
+import AdvancedOptionsTabs from './advanced-options-tabs/advanced-options-tabs';
+
+const labelStyles = css({
+  padding: 0,
+  margin: 0,
+  marginTop: spacing[3],
+});
+
+function AdvancedConnectionOptions() : React.ReactElement {
+  const [open, setOpen] = useState(false)
+  const chevronClass = open ? 'fa-chevron-down' : 'fa-chevron-right'
+  return (
+    <div>
+      <Label htmlFor={''} className={labelStyles}
+        onClick={(e) => setOpen(!open)}
+      >
```

Comment:
Label with an `onClick` element is not a very accessible way to build accordion like component. As leafgreen doesn't provide a component like that out of the box I'd suggest you to look [at this example](https://w3c.github.io/aria-practices/examples/accordion/accordion.html) from WAI-ARIA on how to compose one